javascript/utils.js

```diff
@@ -32,3 +32,9 @@ export const debounce = (callback, delay = 250) => {
     }, delay)
   }
 }
+
+export const extractReflexName = reflexString => {
+  const match = reflexString.match(/(.*->)?(.*)Reflex/)
```

Comment:
Heads-up: we've made the trailing `Reflex` string fragment optional, because it's redundant. See https://github.com/hopsoft/stimulus_reflex/blob/895ee97bcdbe8d80836f177d47a1668be041f61e/lib/stimulus_reflex/channel.rb#L25